services/QuillLMS/db/migrate/20210614205654_create_app_settings.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAppSettings < ActiveRecord::Migration
+  def change
+    create_table :app_settings do |t|
+      t.string :name, null: false
+      t.string :user_whitelist, null: false, default: [], array: true
+      t.boolean :enabled_for_admins, null: false, default: false
+      t.boolean :enabled, null: false, default: false
+      t.integer :pct_active, null: false, default: 0
+
+      t.timestamps null: false
+    end
+    add_index :app_settings, :name
```

Comment:
Yes? Having duplicately-named settings would be very confusing. In addition, it would force access by id rather than name, which is less human friendly.